src/collections/members/aditya-chatterjee/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name:       Aditya Chatterjee
+position: Contributor
+image_path: ./aditya-chatterjee.jpg
+github: warunicorn19
+linkedin: aditya-chatterjee-737729159
+location: Kolkata, India
+bio: 3rd Year CS undergrad of KIIT, Bhubaneswar. Navigating his way through the rapids of the programming world. I joined layer5 in July 2021, 
+    it's been an immense learning elivating not just knowledge but a way of life too.
+badges: - community
```

Comment:
This section can be removed

Reply:
You mean the badges section, or the entire section?

Reply:
The badges section right?

Reply:
Yes 😅

Reply:
hahaha I thought you meant the location and bio too